clients/cloud/kafkacat/README.md

```diff
@@ -0,0 +1,77 @@
+# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using [kafkacat](https://github.com/edenhill/kafkacat).
+
+
+# Prerequisites
+
+* [kafkacat](https://github.com/edenhill/kafkacat) installed on your machine.  You must [build](https://github.com/edenhill/kafkacat#build) `kafkacat` from the latest master branch to get the `-F` functionality that makes it easy to pass in the configuration to your Confluent Cloud configuration file.
+* [Confluent Cloud CLI](https://docs.confluent.io/current/cloud/cli/install.html) installed on your machine. It is provided as part of the Confluent Platform package or may be [installed separately](https://docs.confluent.io/current/cloud/cli/install.html).
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* [Initialize](https://docs.confluent.io/current/cloud/cli/multi-cli.html#connect-ccloud-cli-to-a-cluster) your local Confluent Cloud configuration file using the `ccloud init` command, which creates the file at `$HOME/.ccloud/config`.
+
+
+# Example 1: Hello World!
+
+In this example, the producer writes Kafka data to a topic in Confluent Cloud. 
```

Comment:
"Kafka data"? JSON data

Reply:
Every other README in this example set users the term `Kafka data`, so leaving as-is